src/db/migrations/20200110001514-create-user.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        defaultValue: Sequelize.UUIDV4,
+        primaryKey: true,
+        type: Sequelize.UUID
+      },
+      firstName: {
```

Comment:
Let's be explicit about the "allowNull" property. Use it always, even when the field allows null values. This is for clarity.